setup.py

```diff
@@ -39,6 +39,8 @@
       packages=['jericho'],
       include_package_data=True,
       package_dir={'jericho': 'jericho'},
+      ext_modules=[Extension('jericho.libfrotz', [])],
+      cmdclass = {'build_ext': CustomBuildExt},
       package_data={'jericho': ['libfrotz.so']},
```

Comment:
Remove this

Reply:
We can't. `libfrotz.so` is copied at the right place in the Makefile (https://github.com/Microsoft/jericho/blob/master/frotz/Makefile#L398). I didn't feel like modifying it.

Reply:
Also, because in `CustomBuildExt.build_extension`, I don't call `super().build_extension(ext)` there is no `.so` file being created by setuptools.

Reply:
Ah okay.  I assumed that the extension module would be included in the package automatically by virtue of being listed in `ext_modules`, not due to `build_ext` itself.

Reply:
I did try removing the line and libfrotz.so was indeed missing after pip install the wheel.